src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java

```diff
@@ -30,4 +35,26 @@ void getEmployees_shouldReturnListOfEmployees() {
         assertTrue(response.size() > 0);
     }
 
+    @Test
+    void getSpecification_shouldReturnSpecification() {
+        JobSpecification response = APP.client().target("http://localhost:8080/api/job-specification/1")
+                .request()
+                .get(JobSpecification.class);
+        Assertions.assertNotNull(response);
```

Comment:
I don't think this test adds anything, remove it

Reply:
done